scripts/copy_run_recipe.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+
+import re
+import os
+import sys
+import json
+
+from argparse import ArgumentParser
+from datetime import datetime
+from genologics.lims import Lims
+from genologics.entities import Process
+from genologics.config import BASEURI, USERNAME, PASSWORD
+
+DESC = """EPP for copying run recipe"""
+
+def main(lims, args):
+    log=[]
+    content = None
+    process = Process(lims, id=args.pid)
+    # Read in run recipe file
+    for outart in process.all_outputs():
+        if outart.type == 'ResultFile' and outart.name == 'Run Recipe':
+            try:
+                fid = outart.files[0].id
+                file_name = outart.files[0].original_location
+                content = lims.get_file_contents(id=fid).read()
+            except:
+                raise(RuntimeError("Cannot access the run recipe file."))
+            break
+    # Write file to the server
+    if os.path.exists("/srv/mfs/NovaSeq_data/gls_recipe_novaseq/gls_recipe_ingrid"):
+        try:
+            with open("/srv/mfs/NovaSeq_data/gls_recipe_novaseq/gls_recipe_ingrid/{}".format(file_name), 'w') as sf:
+                sf.write(content)
+                os.chmod("/srv/mfs/NovaSeq_data/gls_recipe_novaseq/gls_recipe_ingrid/{}".format(file_name), 0664)
```

Comment:
I dont think this work here as well (as it dint work in `samplesheet_generator` script). So can remove it